### DISCUSSION TITLE
seed in production env

### DISCUSSION THREAD

**Discussion by ranran2121** [umami-software/umami#1692] [Q&A]
Hi,

sorry for the question but I am not well experienced yet. I need help to solve the following scenario:
I need to populate a table of my DB and I created a seed file. According to the documentation, I understand that 

1. I can run the seed manually (via the `prisma db seed`), which works great
2. or via the prisma migrate dev , which does NOT work if I do not have conflicts. This option works only if I first run `prisma reset` and then `prisma migrate dev`

Apart from this issue, I will then need to apply migration (and seed) in the QA and PROD env, too. And I cannot reset the DB of course . Any suggestion about
- why the `prisma migrate dev` is not running the seed
- how to apply the seed in QA/PROD env?
Thank you

**Comment by briancao** [comment]
What exactly is it that you are trying to seed?  There is only one data point that needs to be created when Umami is initialized, and that's the Admin account (created at the end of migration 01_init).  Otherwise we hold all the logical key/values in the app itself.
